### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daniele-proverbio**
FAQ and credits updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,8 +115,8 @@ The pipeline can be run using any of the following `make` commands:
 
 - Research Luxembourg COVID-19 Taskforce WP6, in the person of Alexander Skupin: supervision and coordination. Contact: alexander.skupin@uni.lu.
 - Daniele Proverbio: R_eff code development, website ideation and content creation
-- [Atte if we include his code]
-- [credits to R3 team for website development and deployment]
+- Atte Aalto: R_t code development
```
Comment (laurentheirendt) [inline README.md]: maybe add the Github handles as well. and I'd say to add the UL, the LCSB, and the Research Luxembourg logos

File: README.md [hunk 2]
```diff
@@ -114,12 +114,14 @@ The pipeline can be run using any of the following `make` commands:
...
-- Daniele Proverbio: R_eff code development, website ideation and content creation
-- Atte Aalto: R_t code development
-- Laurent Heirendt, Jacek Leboida, Christophe Trefois and the LCSB R3 team: docker and website development and deployment
-- [UniLu/LCSB: website hosting and maintenance?]
+- [Daniele Proverbio](https://github.com/daniele-proverbio): R_eff code development, website ideation and content creation
+- [Atte Aalto](https://github.com/AtteAalto): R_t code development
+- [Laurent Heirendt](https://github.com/laurentheirendt), [Jacek Leboida](https://github.com/jLebioda), [Christophe Trefois](https://github.com/trefex) and the LCSB R3 team: docker and website development and deployment
 
-For raised warnings and errors, check the readme section first, for basic troubleshooting.
+   <img src="https://github.com/ResearchLuxembourg/covid-19_reproductionNumber/blob/daniele-proverbio-patch-1/logos/logos.png" alt="logos" width="350"/>
```
Comment (laurentheirendt) [inline README.md]: can you use a relative link here? and put the logos in a `docs/` folder?

Comment (daniele-proverbio) [reply README.md]: So do you prefer to call the folder `docs` instead of `logos`? I can do that

Comment (laurentheirendt) [reply README.md]: yes, as we will likely have more docs at some point.

File: README.md [hunk 3]
```diff
@@ -111,12 +114,14 @@ The pipeline can be run using any of the following `make` commands:
...
 - Research Luxembourg COVID-19 Taskforce WP6, in the person of Alexander Skupin: supervision and coordination. Contact: alexander.skupin@uni.lu.
-- Daniele Proverbio: R_eff code development, website ideation and content creation
-- [Atte if we include his code]
-- [credits to R3 team for website development and deployment]
-- [UniLu/LCSB: website hosting and maintenance?]
+- [Daniele Proverbio](https://github.com/daniele-proverbio): R_eff code development, website ideation and content creation
+- [Atte Aalto](https://github.com/AtteAalto): R_t code development
+- [Laurent Heirendt](https://github.com/laurentheirendt), [Jacek Leboida](https://github.com/jLebioda), [Christophe Trefois](https://github.com/trefex) and the LCSB R3 team: docker and website development and deployment
+
+   <img src="docs/logos.png" alt="logos" width="350"/>
```
Comment (laurentheirendt) [inline README.md]: can you leave the logos as separate files? You can see an example here: https://github.com/LCSB-BioCore/COBREXA.jl/blob/master/README.md

Comment (daniele-proverbio) [reply README.md]: I tried first, but they were aligned badly. What benefit would it bring?

Comment (laurentheirendt) [reply README.md]: The benefit to separate the logos is that they can be used somewhere else or replaced easily 😃

**Comment by laurentheirendt** [comment]
The logos are of very low quality. Can you take the .svg files? You can re-use these: https://github.com/LCSB-BioCore/COBREXA.jl/tree/master/docs/src/assets

**Comment by daniele-proverbio** [comment]
Done, but I could not find an svg for research luxembourg logo. I think the current one is of enough good quality

**Review by laurentheirendt — APPROVED** [review]

**Pull request merged**